composer.json

```diff
@@ -21,7 +21,8 @@
         "symfony-cmf/testing": "^1.3@dev",
         "symfony/dependency-injection": "^2.0.5|3.*",
         "symfony/config": "^2.2|3.*",
-        "symfony/event-dispatcher": "^2.1|3.*"
+        "symfony/event-dispatcher": "^2.1|3.*",
+        "friendsofsymfony/jsrouting-bundle": "@dev"
```

Comment:
Yeah and it's a dep that's not even used by this library. This is one of the problems of Testing 1.0, eventhough this is a component tested without framework integration, many bundles are still automatically loaded by Testing. For Testing 2.0, I would like to remove all config/required bundles from the component, so each library can make choices themselves.